src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -728,6 +727,36 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                IF (InArgs(1) .NE. '*') THEN
+                  isDElightOutVar = .FALSE. ! first assume that it is not a DElight related variable
+                  IF (SameString(InArgs(2)(1:27),'Daylighting Reference Point')) THEN
```

Comment:
You mean I was supposed to follow along all the way to the third item in that comment?  

OK, yeah, that still needs some work.  @JasonGlazer do you have a path forward to make those transition things?  I can take a crack at it tomorrow if needed.